app/e2e/client/wordList.ts

```diff
@@ -0,0 +1,1635 @@
+export const words = [
```

Comment:
what if storing this as a JSON and parsing it when needed? I mean, we're keeping this array in memory even if we don't use it.